gaphor/ui/diagrampage.py

```diff
@@ -370,8 +370,12 @@ def _on_drag_data_received(self, view, context, x, y, data, info, time):
             if not isinstance(
                 element, (UML.Classifier, UML.Package, UML.Property)
             ) or isinstance(element, UML.Association):
-                log.warning(
-                    f"DnD is (temporarily) limited to Classifiers, Packages and Properties, not {type(element).__name__}"
+                self.event_manager.handle(
+                    Notification(
+                        gettext(
+                            "Drag to diagram is (temporarily) limited to Classifiers, Packages and Properties, not {type}."
```

Comment:
Sorry for the nitpicks: could we use an oxford comma? Ideally it would read better if we could made the type plural, but I think it would at least be better if we added an indefinite article "a" before the type.